### PR TITLE
Modif joueur

### PR DESCRIPTION
**Pull request by alexDeschenes**
### Before submitting a pull request, make sure you have:
- [X] Read the [README](https://github.com/TSAP-Laval/acquisition-frontend/blob/master/README.md)
- [X] [Searched](https://github.com/TSAP-Laval/acquisition-frontend/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [X] Reviewed the sample code in [tests](https://github.com/TSAP-Laval/acquisition-frontend/tree/master/tests)

### Purpose of your pull request?
- [ ] Bug fix (encountered problems/errors)
- [X] Feature (request of a new functionality)
- [ ] Enhancement
- [ ] Other

1. The title and description of the feature.
  This feature will allow an administrateur to update a player.

### CODE REVIEW

File: src/components/Manage/Coachs.tsx [hunk 1]
```diff
@@ -3,7 +3,11 @@ import * as $ from "jquery";
 
 import {Button, Alert, Modal} from "react-bootstrap";
 
+<<<<<<< HEAD
```
Comment (laurentlp) [inline src/components/Manage/Coachs.tsx]: Full conflits

File: src/components/Manage/Actions.tsx [hunk 2]
```diff
@@ -4,7 +4,11 @@ import {Button, Alert} from "react-bootstrap";
 import * as $ from "jquery";
 
 import * as requesthandler from './RequestHandler';
+<<<<<<< HEAD
 import actionStore from '../../Stores/ActionsStore';
+=======
+import actionStore from '../../stores/ActionsStore';
+>>>>>>> dev
```
Comment (laurentlp) [inline src/components/Manage/Actions.tsx]: Conflit

**Review by laurentlp — CHANGES_REQUESTED** [review]

File: src/components/Manage/Coachs.tsx (hunk 1 above)
Comment (alexDeschenes) [reply src/components/Manage/Coachs.tsx]: wtf j'ai pas sa moi dans mon code et toute est push

File: src/components/Manage/Actions.tsx [hunk 3]
```diff
@@ -1,137 +1,86 @@
...
+import { serverURL }    from "config"
 
-import * as requesthandler from './RequestHandler';
-<<<<<<< HEAD
-import actionStore from '../../Stores/ActionsStore';
-=======
-import actionStore from '../../stores/ActionsStore';
->>>>>>> dev
+import {Button, Alert} from "react-bootstrap";
+import * as $ from "jquery"
```
Comment (laurentlp) [inline src/components/Manage/Actions.tsx]: PK JQuery ?

**Review by laurentlp — CHANGES_REQUESTED** [review]

File: src/components/Manage/Actions.tsx (hunk 3 above)
Comment (alexDeschenes) [reply src/components/Manage/Actions.tsx]: @mLaribi

**Comment by laurentlp** [comment]
@alexDeschenes  Genre arrange dont la description de ta PR aussi, c'est pas pour rien que j'ai mis un template....

File: src/components/Manage/Coachs.tsx (hunk 1 above)
Comment (laurentlp) [reply src/components/Manage/Coachs.tsx]: Il faut que tu pull !

**Comment by laurentlp** [comment]
Tu as des conflits a cause de mes ajouts faic pull le code et fix les plz @alexDeschenes

File: src/components/Manage/Actions.tsx (hunk 3 above)
Comment (laurentlp) [reply src/components/Manage/Actions.tsx]: Yo @mLaribi

**Comment by laurentlp** [comment]
Arrange la description avant @alexDeschenes

**Comment by laurentlp** [comment]
👍

**Review by laurentlp — APPROVED** [review]

**Pull request merged**